perf_test/batched/KokkosBatched_Test_LU_Host.hpp

```diff
@@ -17,9 +17,10 @@
 #include "KokkosBatched_LU_Decl.hpp"
 #include "KokkosBatched_LU_Serial_Impl.hpp"
 
+#undef __KOKKOSBATCHED_INTEL_MKL_BATCHED__
+
 namespace KokkosBatched {
   namespace Experimental {
-
     namespace PerfTest {
 
 #undef FLOP_MUL
```

Comment:
Can we namespace this ? I know this is not part of this PR, still

Reply:
@srajama1 The PerfTest namespace is still there, only the newline was removed. I added comments to mark end of each namespace as well, hopefully help the readability.